shared/chat/conversation/input.desktop.js

```diff
@@ -2,32 +2,70 @@
 import React, {Component} from 'react'
 import {Box, Icon, Input, Text} from '../../common-adapters'
 import {globalColors, globalMargins, globalStyles} from '../../styles'
+import {Picker} from 'emoji-mart'
 
 import type {Props} from './'
 
-class Conversation extends Component<void, Props, void> {
-  _input: any
+type State = {
+  emojiPickerOpen: boolean,
+  inputText: ?string,
```

Comment:
the '' is just for figuring out selection information, the `Input` tag does get a ?string